vaadin-date-time-picker-flow/src/test/java/com/vaadin/flow/component/datetimepicker/DateTimePickerTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2000-2019 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datetimepicker;
+
+import java.time.LocalDateTime;
+
+import net.jcip.annotations.NotThreadSafe;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.vaadin.flow.component.UI;
+
+import static org.junit.Assert.assertEquals;
+
+@NotThreadSafe
+public class DateTimePickerTest {
+
+    private UI ui;
```

Comment:
All the UI mocking seems to be redundant so far. Feel free to keep it if you think it's needed later though.

Reply:
Honestly I have no idea what this part does. :D I can remove it if it works without.

Reply:
It seems that this UI needs to be created here, otherwise all tests fail pointing to an NPE at `DatePicker` constructor here: https://github.com/vaadin/vaadin-date-picker-flow/blob/master/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java#L88

where it calls `UI.getCurrent().getLocale()`.

Reply:
Ok, that explains it